docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -108,15 +124,21 @@ The classic (original) way to use roles is with the ``roles`` option for a given
         - common
         - webservers
 
-When you use the ``roles`` option at the play level, for each role 'x':
+When you use the ``roles`` option at the play level, each role 'x' looks for a ``main.yml/main.yaml/main`` by default (you can specify otherwise with ``include/import_role``) in the following directories:
```

Comment:
with `roles:` it is not a default, but the only way it works, I would rephrase to avoid confusion as the 'default' applies only to include/import_role.

Reply:
Would rephrasing it like this be appropriate?

```
When you use the ``roles`` option at the play level, each role 'x' looks for a ``main.yml/main.yaml/main`` file (if you use ``include_role/import_role``, you can specify a different file name) in the following directories:
```

This way I would remove the confusion and still keep in the relevant information. I was also thinking that I could move the part in the parentheses into this note:
```
.. note::
    The ``meta`` directory is an exception because it does not allow for customization.
```
The result would look like this:
```
.. note::
    If you use ``include_role/import_role``, you can specify a different file name. The ``meta`` directory is an exception because it does not allow for customization.    
```